Sources/NIO/EventLoopFuture.swift

```diff
@@ -997,7 +1011,7 @@ extension EventLoopFuture {
     /// - Parameter futures: An array of homogenous `EventLoopFuture`s to gather results from.
     /// - Returns: A new `EventLoopFuture` with all the results of the provided futures.
     public static func whenAllComplete(_ futures: [EventLoopFuture<Value>],
-                                       eventLoop: EventLoop) -> EventLoopFuture<[Result<Value, Error>]> {
+                                       on eventLoop: EventLoop) -> EventLoopFuture<[Result<Value, Error>]> {
```

Comment:
Sorry, just noticed this: this label change needs a note in the migration notes in `docs` and also a helper in `_NIO1APIShims`?

Reply:
`whenAllComplete` doesn't exist in NIO 1 - this was introduced directly to NIO 2 because of its use of `Result<Value, Error>`

See #773

Reply:
Ah, you're quite right.